js/bybit.js

```diff
@@ -3572,88 +4822,21 @@ module.exports = class bybit extends Exchange {
             throw new NotSupported (this.id + ' editOrder() does not support spot markets');
         } else if (isUsdcSettled) {
             return await this.editUsdcOrder (id, symbol, type, side, amount, price, params);
-        } else {
-            return await this.editContractOrder (id, symbol, type, side, amount, price, params);
         }
+        return await this.editContractOrder (id, symbol, type, side, amount, price, params);
```

Comment:
Do we need to update `editOrder` to add the unifiedMargin part? 
https://bybit-exchange.github.io/docs/derivativesV3/unified_margin/#t-dv_replaceorder